presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -547,15 +547,15 @@ void PrestoTask::updateMemoryInfoLocked(
   protocol::TaskStats& prestoTaskStats = info.stats;
 
   const auto veloxTaskMemStats = task->pool()->stats();
-  prestoTaskStats.userMemoryReservationInBytes = veloxTaskMemStats.currentBytes;
+  const auto currentBytes = task->pool()->usedBytes();
+  prestoTaskStats.userMemoryReservationInBytes = currentBytes;
```

Comment:
Could we get 'currentBytes' from 'veloxTaskMemStats'?
they are both received from the pool.

Reply:
stats() still use currentBytes(), and I will switch to use usedBytes in stats() after deprecate currentBytes.